test/poodinis/autowiretest.d

```diff
@@ -9,6 +9,7 @@ import poodinis;
 import poodinis.test.testClasses;
 
 import std.exception;
+import std.meta;
```

Comment:
Why do you need std.meta here?

Reply:
It's to import `staticIndexOf`, which is used to determine if a type has been previously been seen and imported in `createImportsString`. This could be changed to be `import std.meta : staticIndexOf` to make it clear which symbols are being imported from this module.

This style could be added to the generated import statements to only import the symbols being used and avoid any potential naming conflicts e.g. `string imports = "import " ~ moduleName!Type ~ " : " ~ Type.stringof ~ ";";`

Reply:
But this is the test, why here? I understand why you would need it in imports.d